Hec.Cwms/CwmsDatabase.cs

```diff
@@ -17,6 +17,8 @@ public class CwmsDatabase
     public CwmsDatabase(Oracle oracle)
     {
       this.oracle = oracle;
+      this.officeID = oracle.officeid;
```

Comment:
I don't think Oracle should have an officeID.   Perhaps the CwmsDatabase() constructor should take an office id?  Then the setOfficeID can be private?

Reply:
Oops. Yes agreed. Got officeid out of Oracle and into CwmsDatabase constructor. Also set SetOffice() to private.